src/pheval/prepare/create_noisy_phenopackets.py

```diff
@@ -121,15 +121,16 @@ def convert_patient_terms_to_parent(
         parent_terms = []
         for term in hpo_terms_to_be_changed:
             try:
-                parent_terms.append(
-                    self.retrieve_hpo_term(self.hpo_ontology.hierarchical_parents(term.type.id)[0])
-                )
+                parents = self.hpo_ontology.hierarchical_parents(term.type.id)
+                parent_terms.append(self.retrieve_hpo_term(random.choice(parents)))
```

Comment:
What could throw the error here? Is it being thrown because the term is obsolete? If so, why not just check if the term is obsolete first,  handle that and then just follow the logic you have in the exception. This way, no exception checking required.